### PR TITLE
Update Monocle to 8u60_b19 (a2bb669fd477)

### PR DESCRIPTION
**Pull request by brcolow**
This updates the current repo to 8u60_b19. If you could create a new branch on the main TestFX/Monocle repo (perhaps named 8u60_b19) and do a release so that the artifact is available on Maven Central as it is now, it would really be awesome.

### CODE REVIEW

**Comment by hastebrot** [comment]
Whoa, didn't saw that. Thanks!

**Comment by RobzInadE** [comment]
Bump! Please release this (and also u40 if possible) so they're available on maven central.

Regards

**Comment by hastebrot** [comment]
I'll look into this tomorrow.

File: gradle.properties [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-version = 1.8.0_20
+version = 1.8.0_60_b19
 
 description = Headless graphics driver for JavaFX.
 vendor = The OpenJDK Community
 
-url = https://github.com/TestFX/Monocle
-sourceUrl = https://github.com/TestFX/Monocle
-issuesUrl = https://github.com/TestFX/Monocle/issues
+url = https://github.com/brcolow/Monocle
```
Comment (hastebrot) [inline gradle.properties]: Revert `url`, `sourceUrl` and `issuesUrl`.

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,5 @@
 *.ipr
 *.iws
 *.cmd
+.DS_Store
+tags
```
Comment (hastebrot) [inline .gitignore]: Seems the `tags` file slipped into this PR despite the ignore line.

**Comment by brcolow** [comment]
Do you want me to clean this up?

**Comment by hastebrot** [comment]
@brcolow That would be nice.

**Comment by brcolow** [comment]
Sorry for the delay, here are the fixes you asked for:

Ref: https://github.com/TestFX/Monocle/pull/10

**Comment by hastebrot** [comment]
Ok, thanks. My plan is to release the artifacts as e.g. `testfx-monocle-2015XXXX-1.8.0_20.jar`.

**Comment by hastebrot** [comment]
~~~java
public class Version {
    public static void main(String[] args) {
        System.out.println(System.getProperty("java.version"));
        System.out.println(System.getProperty("java.runtime.version"));
    }
}
~~~

~~~
1.8.0_72
1.8.0_72-b15
~~~

~~~
9-ea
9-ea+106-jigsaw-nightly-h4488-20160219
~~~

**Pull request closed without merging**